doc/conditional_independence.rst

```diff
@@ -0,0 +1,180 @@
+.. _conditional_independence:
+
+========================
+Conditional Independence
+========================
+
+.. currentmodule:: dodiscover.ci
+
+Testing for invariances in the data, such as conditional independence (CI) can be represented graphically
+in the form of d-separation statements under the causal faithfulness assumption. Given this, one
+is interested in high-powered and well-controlled CI tests that can be used to test for CI in data.
```

Comment:
While it is a good fit for dodiscover, I wouldn't make any causal or general graph reference at this point, since (conditional) independence tests are just hypothesis tests as mentioned in the next paragraph. However, it fits to the dodiscover story of course, but if we move it to the stats package, this part should probably be re-written/removed.

Maybe you can move this graphical remark to the constrained based causal discovery part?

Reply:
Done. Lmk wdyt.